CourseManager/Courses/Course 1 Build a SwiftUI app for iOS 15/Model/Course.swift

```diff
@@ -24,7 +24,7 @@ var featuredCourses = [
     Course(title: "React Hooks Advanced", subtitle: "20 sections - 3 hours", text: "Learn how to build a website with Typescript, Hooks, Contentful and Gatsby Cloud", image: "Illustration 2", background: "Background 2", logo: "Logo 3")
 ]
 
-var courses = [
+var `courses` = [
```

Comment:
@sghourab it's not good practice. Static public variables better to move in a separate Structures: 

```
struct Course {

 struct MockData {
 
   static let featured = [...]
   static let courses = [....]
 }
}

```